docs/source/examples/move_and_copy.rst

```diff
@@ -0,0 +1,86 @@
+###############
+ Move And Copy
+###############
+
+This topic describes TensorBay dataset operations:
+
+- :ref:`examples/move_and_copy:Copy Segment`
+- :ref:`examples/move_and_copy:Move Segment`
+- :ref:`examples/move_and_copy:Copy Data`
+- :ref:`examples/move_and_copy:Move Data`
+
+Take the `Oxford-IIIT Pet <https://gas.graviti.cn/dataset/data-decorators/OxfordIIITPet>`_
+as an example. Its structure looks like::
+
+    datasets/
+        test/
+            Abyssinian_002.jpg
+            ...
+        trainval/
+            Abyssinian_001.jpg
+            ...
+
+.. note::
+
+   Before operating this dataset,
+   `fork <https://gas.graviti.cn/dataset/data-decorators/OxfordIIITPet>`_ it first.
+
+Get the dataset client.
+
+.. literalinclude:: ../../../docs/code/move_and_copy.py
+   :language: python
+   :start-after: """Get Dataset Client"""
+   :end-before: """"""
+
+
```

Comment:
Please add this line before L35
There are currently two segments: ``test`` and ``trainval``.